docs/tutorials/resource-migration.md

```diff
@@ -138,98 +110,92 @@ spec:
     clusterAffinity:
       clusterNames:
       - member1
-  priority: 0
+      - member2
```

Comment:
In the migration process, it is not recommended to change the application topology. 

I don't think `member2` should be present here.
And the following content should be updated accordingly.